tests/Api/CachedIdentifiersExtractorTest.php

```diff
@@ -45,7 +45,9 @@ public function testFirstPass()
 
         $identifiersExtractor = new CachedIdentifiersExtractor($cacheItemPool->reveal(), $decoration->reveal(), null);
 
-        $this->assertEquals(['id' => 1], $identifiersExtractor->getIdentifiersFromItem($dummy));
+        $expectedResult = ['id' => 1];
+        $this->assertEquals($expectedResult, $identifiersExtractor->getIdentifiersFromItem($dummy));
+        $this->assertEquals($expectedResult, $identifiersExtractor->getIdentifiersFromItem($dummy), 'Trigger the local cache');
```

Comment:
Good catch. `testSecondPass` should be update to bypass the local cache IMO (by creating a new instance).